src/server/HttpHandler.ts

```diff
@@ -20,17 +20,23 @@ export class HttpHandler implements LambdaHttpHandler {
             };
         }
         let body: any = undefined;
-        const req = JSON.parse(request.body) as JsonRpcRequest;
+        const req = JSON.parse(JSON.stringify(request.body)) as JsonRpcRequest;
```

Comment:
Can just use `request.body as JsonRpcRequest`

Reply:
@naiemk i tried this and it doesn't work because request.body comes as a string containing an object i.e "{ body: {....} }" and this cannot be cast to `JsonRpcRequest`.